### PR TITLE
Move base styles to TypeScript

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️

### CODE REVIEW

File: common/views/themes/base/inline-fonts.ts [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
-https://unicode.org/charts/PDF/U0000.pdf
-U+0020: space,
-U+0022: quotation,
-U+002C: comma,
-U+002E: fullstop,
-U+0030-0039 : digits 0-9
-U+003F: question mark,
-U+0041-005A: uppercase,
-U+0061-007A: lowercase
-*/
```
Comment (jamesgorrie) [inline common/views/themes/base/inline-fonts.ts:19]: This still seems to be useful information?

File: common/views/themes/base/wellcome-normalize.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { themeValues } from '../config';
+
+export const wellcomeNormalize = `
```
Comment (jamesgorrie) [inline common/views/themes/base/wellcome-normalize.ts:3]: Any reason to not just have normalize?

**Review by jamesgorrie — APPROVED** [review]

File: common/views/themes/base/wellcome-normalize.ts (hunk 2 above)
Comment (davidpmccormick) [reply common/views/themes/base/wellcome-normalize.ts:3]: we also have normalize as a vendor asset (still in scss at this point), but these were our own additions to that. I think keeping the two separate probably makes sense?

Comment (jamesgorrie) [reply common/views/themes/base/wellcome-normalize.ts:3]: Makes sense, could call it `extendedNormalize` or something, but nitpicking here.

Comment (jamesgorrie) [reply common/views/themes/base/wellcome-normalize.ts:3]: Assuming we could just call this normalise, pull in the vendor version and extend it?

**Pull request merged**